### DISCUSSION TITLE
Yi Home 1080p AI+ Support - SFUS, 12.2.0.2*

### DISCUSSION THREAD

**Discussion by Pragtically** [alienatedsec/yi-hack-v5#306] [Q&A]
Can't seem to find this specific Serial/Firmware combination. Is this supported/in development? Thanks!

**Comment by stkstv** [comment]
I would also like an answer to this. i am about to lose my mind trying to find a cheaper alternative for homekit cameras. Everything i get ends up not working because of firmware or its the right model but different firmware.
